lib/agents/jawsm.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+const runtimePath = require('../runtime-path');
+const ConsoleAgent = require('../ConsoleAgent');
+
+class JawsmAgent extends ConsoleAgent {
+  constructor(options) {
+    super(options);
+
+    // disable node error coloring stuff
+    this.cpOptions = {
+      env: {
+        NO_COLOR: 1
+      }
+    };
```

Comment:
why is this here?

Reply:
@CanadaHonk do you remember the reason for https://github.com/test262-fyi/eshost/commit/b85860cb22d3120a9ab8789e3e6fa4f5c06f50b8?

Reply:
ok, actually, I was wrong, this doesn't really affect how many tests pass for me, so I'll remove it